### PR TITLE
Move plugin URL to github

### PR DESCRIPTION
**Pull request by bitwiseman**
* JENKINS issue(s):
    * Moving documentation and changelog to github
@dwnusbaum @car-roll @kshultzCB @olamy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,31 @@
-A config-like syntax for defining Pipelines. See [the
-docs on jenkins.io](https://jenkins.io/doc/)
-for more information on how to use it.
+# Declarative Jenkins Pipelines
+
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/pipeline-modeldefinition)](https://plugins.jenkins.io/pipeline-modeldefinition-plugin)
```
Comment (car-roll) [inline README.md]:
```suggestion
[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/pipeline-model-definition)](https://plugins.jenkins.io/pipeline-model-definition)
```

File: README.md [hunk 2]
```diff
@@ -1,10 +1,31 @@
...
+[![Changelog](https://img.shields.io/github/v/tag/jenkinsci/pipeline-modeldefinition-plugin?label=changelog)](https://github.com/jenkinsci/pipeline-modeldefinition-plugin/blob/master/CHANGELOG.md)
```
Comment (car-roll) [inline README.md]:
```suggestion
[![Changelog](https://img.shields.io/github/v/tag/jenkinsci/pipeline-model-definition-plugin?label=changelog)](https://github.com/jenkinsci/pipeline-model-definition-plugin/blob/master/CHANGELOG.md)
```

File: README.md [hunk 3]
```diff
@@ -1,10 +1,31 @@
...
+[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/pipeline-modeldefinition?color=blue)](https://plugins.jenkins.io/pipeline-modeldefinition)
```
Comment (car-roll) [inline README.md]:
```suggestion
[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/pipeline-model-definition?color=blue)](https://plugins.jenkins.io/pipeline-model-definition)
```

File: README.md [hunk 4]
```diff
@@ -1,10 +1,31 @@
...
+**WARNING**:
+Older versions of this plugin may not be safe to use. Please review the
+following warnings before using an older version:
+
+-   [Script Security sandbox
+    bypass](https://jenkins.io/security/advisory/2019-01-08/#SECURITY-1266)
+
+## Documentation
+
+* [Changelog](https://github.com/jenkinsci/pipeline-modeldefinition-plugin/blob/master/CHANGELOG.md)
```
Comment (car-roll) [inline README.md]:
```suggestion
* [Changelog](https://github.com/jenkinsci/pipeline-model-definition-plugin/blob/master/CHANGELOG.md)
```

File: README.md [hunk 5]
```diff
@@ -1,10 +1,31 @@
...
+* [Developer Guide](https://github.com/jenkinsci/pipeline-modeldefinition-plugin/blob/master/DEV_GUIDE.md)
```
Comment (car-roll) [inline README.md]:
```suggestion
* [Developer Guide](https://github.com/jenkinsci/pipeline-model-definition-plugin/blob/master/DEV_GUIDE.md)
```

File: README.md [hunk 6]
```diff
@@ -1,10 +1,31 @@
...
+* [Instructions for extending this plugin](https://github.com/jenkinsci/pipeline-modeldefinition-plugin/blob/master/EXTENDING.md)
```
Comment (car-roll) [inline README.md]:
```suggestion
* [Instructions for extending this plugin](https://github.com/jenkinsci/pipeline-model-definition-plugin/blob/master/EXTENDING.md)
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -0,0 +1,755 @@
...
+            Pre-existing environment variables, such as `PATH`, can be
+            overridden in the `environment` directive.
+        -   [JENKINS-44034](https://issues.jenkins-ci.org/browse/JENKINS-44034) -
+            Variable references in `environment` variable definitions no
+            longer require curly braces.
+        -   [JENKINS-46112](https://issues.jenkins-ci.org/browse/JENKINS-46112) -
+            Logs for `error` steps visible in Blue Ocean correctly.
+        -   [JENKINS-45991](https://issues.jenkins-ci.org/browse/JENKINS-45991) - `environment` variables
+            can be configured to have default values,
+            i.e., `FOO = FOO ?: "default"```
```
Comment (dwnusbaum) [inline CHANGELOG.md]: Something is wrong with the formatting here, not sure exactly what. Here's my guess:

```suggestion
            i.e., `FOO = FOO ?: "default"`
```

**Pull request merged**